.github/workflows/ci_cd.yml

```diff
@@ -766,3 +769,94 @@ jobs:
     with:
       tag: ${{ needs.get-image-tag.outputs.image_tag }}
     secrets: inherit
+
+  ################
+  # Notification #
+  ################
+
+  send_report:
+    name: Send Slack report
+    runs-on: ubuntu-latest
+    if: |
+      !cancelled() &&
+      github.event_name == 'push' &&
+      github.repository == 'WordPress/openverse' &&
+      (
+        ((needs.get-changes.outputs.frontend == 'true' || needs.get-changes.outputs.documentation == 'true') && needs.emit-docs.result != 'success') ||
+        (needs.determine-images.outputs.do_publish == 'true' && needs.publish-images.result != 'success') ||
+        (needs.get-changes.outputs.frontend == 'true' && needs.deploy-frontend.result != 'success') ||
+        (needs.get-changes.outputs.api == 'true' && needs.deploy-api.result != 'success')
```

Comment:
This links the condition and the expected outcome and the report will be sent in case there is a mismatch.

- If the frontend or docs have changed, new docs must be published.
- If there are any images to publish, the `publish-images` job should successfully publish them.
- If the frontend or the API have changes, they should be deployed.

Each condition expects a successful outcome of a particular job and if not, the report will be sent.

Reply:
This means that if you see the report in Slack, something's not right and calls for an investigation.

Reply:
Nice! This makes a lot of sense to me, thanks for digging into this :pray: Having the get-changes explicitly next to each one also makes it easier to understand how the conditions work and the intention of other jobs too. Like the frontend one: are there frontend changes? If so, then these things should have run and been successful. If they weren't then something is wrong.

This is very clear, thank you, again!